src/examples/turtlebot4/README.md

```diff
@@ -0,0 +1,138 @@
+# Turtlebot4 tutorial
+
+The following tutorial will be presented during [ROSCon 2024](https://roscon.ros.org/2024/)
```

Comment:
This is less interesting information as the tutorial is to have longevity beyond ROSCon, and once guided here, people will already know.

Reply:
fixed